hack/create-user-workspace.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
+
+if [[ -z ${1} ]]
+then
+  echo "You have to provide the name of the service you want to initiate in the user workspace:"
+  echo "     create-user-workspace.sh <appstudio|hacbs>"
+  exit 1
+fi
+
+SERVICE_NAME=${1}
+
+ROOT_WORKSPACE=${ROOT_WORKSPACE:-"root"}
+APPSTUDIO_WORKSPACE=${APPSTUDIO_WORKSPACE:-"redhat-appstudio"}
+HACBS_WORKSPACE=${HACBS_WORKSPACE:-"redhat-hacbs"}
+
+if [ "${ROOT_WORKSPACE}" == "~" ]; then
+  ROOT_WORKSPACE=$(KUBECONFIG=${KCP_KUBECONFIG} kubectl ws . | cut -f2 -d'"')
+fi
+
+APPSTUDIO_SP_WORKSPACE=${APPSTUDIO_SP_WORKSPACE:-${ROOT_WORKSPACE}:${APPSTUDIO_WORKSPACE}}
+HACBS_SP_WORKSPACE=${HACBS_SP_WORKSPACE:-${ROOT_WORKSPACE}:${HACBS_WORKSPACE}}
+
+USER_APPSTUDIO_WORKSPACE=${USER_APPSTUDIO_WORKSPACE:-"${SERVICE_NAME}"}
+echo "Accessing the home workspace:"
+kubectl ws '~'
+echo "Creating & accessing AppStudio workspace '${USER_APPSTUDIO_WORKSPACE}':"
+kubectl ws create ${USER_APPSTUDIO_WORKSPACE}  --ignore-existing --type root:universal --enter
+
+kustomize build ${ROOT}/apibindings/${SERVICE_NAME}/ | sed "s|\${APPSTUDIO_SP_WORKSPACE}|${APPSTUDIO_SP_WORKSPACE}|g;s|\${HACBS_SP_WORKSPACE}|${HACBS_SP_WORKSPACE}|g" | \
```

Comment:
use rather `kubectl kustomize` to not require `kustomize` on the user system.

Reply:
cool, I didn't know that it can be used in this way aeeac78